content/release-10-18+/devices-10-18+.md

```diff
@@ -33,6 +33,14 @@ Build version</th>
 </tr>
 </thead><tbody>
 
+<tr>
+<td>Device management</td>
+<td>If a pending device in the registration list has been accepted, the device will now be removed from the list - no "accepted" status will be shown.</td>
```

Comment:
```suggestion
<td>If a pending device in the registration list has been accepted, the device is now removed from the list instead of the list showing the "accepted" status for the device.</td>
```